release_notes_generator.py

```diff
@@ -0,0 +1,408 @@
+from __future__ import print_function
```

Comment:
should this file be in content root?

Reply:
the old release notes generator was there, do you suggest something else?
maybe `Utils`?

Reply:
yea